simulariumio/cytosim/cytosim_converter.py

```diff
@@ -254,6 +255,20 @@ def _read(input_data: CytosimData) -> TrajectoryData:
                 agent_data,
                 uids,
             )
+        # get display data (geometry and color)
+        for object_type in input_data.object_info:
+            for tid in input_data.object_info[object_type].display_data:
+                display_data = input_data.object_info[object_type].display_data[tid]
+                if "fiber" in object_type and (
+                    display_data.display_type != DISPLAY_TYPE.NONE
+                    and display_data.display_type != DISPLAY_TYPE.FIBER
+                ):
+                    display_data.display_type = DISPLAY_TYPE.FIBER
+                    print(
+                        f"{display_data.name} display type of "
+                        f"{display_data.display_type} was changed to FIBER"
```

Comment:
this needs to print the stringified version of the display_type.  (see if the enum can be modified to hold the string values maybe!)